packages/truffle-compile/compilerSupplier/loadingStrategies/LoadingStrategy.js

```diff
@@ -15,11 +15,17 @@ class LoadingStrategy {
     };
     this.config = Object.assign({}, defaultConfig, options);
 
-    const compilerCachePath = path.resolve(
+    const compilersDir = path.resolve(
```

Comment:
I would say just get rid of this variable name as we don't need it. That also lets you get rid of one of the "if" statements below. All we really want to ensure is that whatever cacheing directory we choose to use exists.

Reply:
I take it back. I guess we do need both to ensure both the outer and inner directory exists.

Reply:
Yeah, it's a bummer.